fact_extractor/plugins/unpacking/tpl/test/test_tp_wr702n.py

```diff
@@ -67,6 +77,13 @@ def test_extraction(self):
         expected_fs = 1
         expected_remaining = 2
 
-        self.assertEqual(len(files),
-                         expected_remaining + expected_container_header + expected_bootloader + expected_main + expected_os + expected_fs + expected_img0_header,
-                         'file number incorrect')
+        assert (
+            len(files)
+            == expected_remaining
+            + expected_container_header
```

Comment:
Wired formatting, did black do this?

Reply:
Yes and I also do not know why :sweat_smile: